### PR TITLE
Document message templates and generated value

### PR DESCRIPTION
**Pull request by barmac**
## Description

This pull request documents the message templates and generated values which were added in 8.3 as part of https://github.com/camunda/product-hub/issues/817

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

Closes https://github.com/bpmn-io/internal-docs/issues/797

### CODE REVIEW

**Review by christinaausley — DISMISSED** [review]

File: docs/components/modeler/desktop-modeler/element-templates/defining-templates.md [hunk 1]
```diff
@@ -331,6 +348,26 @@ Configures the [task type](../../../bpmn/service-tasks/#task-definition) for a s
 
 The `zeebe:property` binding allows you to set any arbitrary property for an outside system. It does not impact execution of the Zeebe engine.
 
+#### `bpmn:Message#property`
+
+| **Binding `type`**          | `bpmn:Message#property`                            |
+| --------------------------- | -------------------------------------------------- |
+| **Valid property `type`'s** | `String`<br />`Text`<br />`Hidden`<br />`Dropdown` |
+| **Binding parameters**      | `name`: The name of the property                   |
+| **Mapping result**          | `<bpmn:Message [name]="[userInput]" />`            |
```
Comment (philippfromme) [inline docs/components/modeler/desktop-modeler/element-templates/defining-templates.md:357]:
```suggestion
| **Mapping result**          | `<bpmn:message [name]="[userInput]" />`            |
```

Comment (philippfromme) [reply docs/components/modeler/desktop-modeler/element-templates/defining-templates.md:357]: We should be consistent here. XML tag names are lowercase, right?

**Review by philippfromme — CHANGES_REQUESTED** [review]
Maybe we can also remove the superfluos spaces in the other bindings (e.g. https://github.com/camunda/camunda-docs/pull/2923/files#diff-f537878378043ecc2e72d120ff3b9bec1edb4f1bae0637750526fb821feb411fR335).

File: docs/components/modeler/desktop-modeler/element-templates/defining-templates.md (hunk 1 above)
Comment (barmac) [reply docs/components/modeler/desktop-modeler/element-templates/defining-templates.md:357]: Correct.

**Comment by barmac** [comment]
> Maybe we can also remove the superfluos spaces in the other bindings (e.g. https://github.com/camunda/camunda-docs/pull/2923/files#diff-f537878378043ecc2e72d120ff3b9bec1edb4f1bae0637750526fb821feb411fR335).

What do you mean?

**Review by christinaausley — APPROVED** [review]

**Review by philippfromme — APPROVED** [review]
✅

**Pull request merged**